ports/esp32/boards/LOLIN_S2_MINI/modules/s2mini.py

```diff
@@ -0,0 +1,52 @@
+# LOLIN S2 MINI MicroPython Helper Library
+
+
+from micropython import const
+from machine import Pin, SPI, ADC
+import machine, time
+
+import network
+
+# Pin Assignments
+
+
+# SPI
+SPI_MOSI = const(11)
+SPI_MISO = const(9)
+SPI_CLK = const(7)
+
+# I2C
+I2C_SDA = const(33)
+I2C_SCL = const(35)
+
+# DAC
+DAC1 = const(17)
+DAC2 = const(18)
+
+# LED
+LED = const(15)
+
+# BUTTON
+BUTTON = const(0)
+
+
+class led:
+    def __init__(self):
+        self._led = Pin(LED, Pin.OUT, value=0)
+
+    def on(self):
+        self._led.value(1)
+
+    def off(self):
+        self._led.value(0)
+
+
+class button:
```

Comment:
As with the LED class I think it's much simpler to just do:
```python
button = Pin(BUTTON, Pin.IN, Pin.PULL_UP)
```
and not have this class at all.